ports/servoshell/webview.rs

```diff
@@ -542,14 +564,28 @@ where
                     };
                 },
                 EmbedderMsg::WebViewOpened(new_webview_id) => {
-                    self.webviews.insert(new_webview_id, WebView {});
+                    let scale = self.window.hidpi_factor().get();
+                    let toolbar = self.window.toolbar_height().get();
+
+                    // Adjust for our toolbar height.
+                    // TODO adjust for egui window decorations if we end up using those
```

Comment:
```suggestion
                    // TODO: Adjust for egui window decorations if we end up using those.
```

Reply:
Fixed in [8c01639](https://github.com/servo/servo/pull/31417/commits/8c01639b1993c3b6c5fa57c1fcfc412716cf6f05)